### PR TITLE
Add link to MySQL docs for max_packet_size diagnostic warning

### LINKED ISSUE #20970: In the Diagnostics plugin (core) add a helpful link to Packet Too Large for max_allowed_packet
A user has suggested we add a useful footnote in our comments:

> In Settings page ?page=matomo-systemreport maybe you could link max_allowed_packet pointing tohttps://dev.mysql.com/doc/refman/8.0/en/packet-too-large.html where one can find how to fix it.

### PR DESCRIPTION
**Pull request by bx80**
### Description:

Fixes #20970 

This PR simply adds a link to the MySQL docs for the `max_packet_size` diagnostic warning.

Translation file were updated because this adds two placeholders for the link, existing placeholder were renumbered but no language changes were made.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by michalkleiner** [comment]
Does the same docs apply to different versions of MySQL? That's the only reason why I didn't pick this up as I wasn't sure whether we should e.g. link to an FAQ page with different links for different MySQL versions.

**Comment by bx80** [comment]
I did think about having different links depending on the database engine, but we don't have any proper system for detecting different database types and this config setting is exactrly the same for MySQL 5.x, 8.x and MariaDb, so the MySQL 8.x documentation page will work for all. 

Later on when we implement more database abstraction then this link could be provided by the database driver.

File: plugins/Diagnostics/Diagnostic/DbMaxPacket.php [hunk 1]
```diff
@@ -43,7 +43,9 @@ public function execute()
             $formatter = new Formatter\Html();
             $pretty = $formatter->getPrettySizeFromBytes($maxPacketBytes['Value'], 'M', $precision = 1);
             $configured = str_replace(array(' M', '&nbsp;M'), 'MB', $pretty);
-            $comment = Piwik::translate('Diagnostics_MysqlMaxPacketSizeWarning', array('64MB', $configured));
+            $comment = Piwik::translate('Diagnostics_MysqlMaxPacketSizeWarning',
+                ['<a href="https://dev.mysql.com/doc/refman/8.0/en/packet-too-large.html" rel="noreferrer noopener" target="_blank">',
```
Comment (michalkleiner) [inline plugins/Diagnostics/Diagnostic/DbMaxPacket.php]: We can use version agnostic link here and it redirects to the latest one automatically
```suggestion
                ['<a href="https://dev.mysql.com/doc/refman/en/packet-too-large.html" rel="noreferrer noopener" target="_blank">',
```

Comment (bx80) [reply plugins/Diagnostics/Diagnostic/DbMaxPacket.php]: Nice! :+1:

**Pull request merged**